It appears upstream forgot to update the project version in the build
system when tagging the release.  Alter the pom.xml files to allow the
proper version to be easily substituted in.

--- pom.xml.orig	2025-09-21 09:12:19
+++ pom.xml	2026-08-31 19:18:50
@@ -7,7 +7,7 @@
 
     <groupId>org.jmock</groupId>
     <artifactId>jmock-parent</artifactId>
-    <version>2.13.2-SNAPSHOT</version>
+    <version>@@VERSION@@</version>
     <packaging>pom</packaging>
     <name>jMock 2 Parent</name>
 
--- jmock/pom.xml.orig	2025-09-21 09:12:19
+++ jmock/pom.xml	2026-08-31 19:30:07
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
--- jmock-example/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-example/pom.xml	2026-08-31 19:30:40
@@ -12,7 +12,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
--- jmock-imposters/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-imposters/pom.xml	2026-08-31 19:32:13
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
     <artifactId>jmock-imposters</artifactId>
     <description>Class mocks are more numerous than interface mocks, so drop the legacy name</description>
@@ -36,4 +36,4 @@
         </plugins>
     </build>
 
-</project>
\ No newline at end of file
+</project>
--- jmock-imposters-testdata/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-imposters-testdata/pom.xml	2026-08-31 19:32:47
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
 
     <artifactId>jmock-imposters-testdata</artifactId>
@@ -40,4 +40,4 @@
             </plugin>
         </plugins>
     </build>
-</project>
\ No newline at end of file
+</project>
--- jmock-imposters-tests/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-imposters-tests/pom.xml	2026-08-31 19:33:15
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
     <artifactId>jmock-imposters-tests</artifactId>
 
@@ -94,4 +94,4 @@
         </profile>
     </profiles>
 
-</project>
\ No newline at end of file
+</project>
--- jmock-junit3/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-junit3/pom.xml	2026-08-31 19:33:56
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
 
     <artifactId>jmock-junit3</artifactId>
--- jmock-junit4/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-junit4/pom.xml	2026-08-31 19:34:20
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
 
     <properties>
--- jmock-junit5/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-junit5/pom.xml	2026-08-31 19:34:39
@@ -9,7 +9,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
 
     <properties>
--- jmock-legacy/pom.xml.orig	2025-09-21 09:12:19
+++ jmock-legacy/pom.xml	2026-08-31 19:35:01
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
     </parent>
 
     <properties>
--- testjar/pom.xml.orig	2025-09-21 09:12:19
+++ testjar/pom.xml	2026-08-31 19:35:39
@@ -11,7 +11,7 @@
     <parent>
         <groupId>org.jmock</groupId>
         <artifactId>jmock-parent</artifactId>
-        <version>2.13.2-SNAPSHOT</version>
+        <version>@@VERSION@@</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
